GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SampleViewManager.java

```diff
@@ -64,15 +75,23 @@ public void onInit(GVRContext gvrContext) {
         GVRCylinderSceneObject cylinderObject = new GVRCylinderSceneObject(
                 gvrContext);
         GVRConeSceneObject coneObject = new GVRConeSceneObject(gvrContext);
+        GVRWebViewSceneObject webViewObject = createWebViewObject(gvrContext);
+        GVRCameraSceneObject cameraObject = new GVRCameraSceneObject(
+                gvrContext, 8.0f, 4.0f,
+                ((SceneObjectActivity) mActivity).getCamera());
```

Comment:
It was late at night.  What you suggest is certainly much cleaner.  Just made that change in my latest patchset.